README.md

```diff
@@ -23,8 +23,8 @@ The autograding effort is broken into three main parts:
 ## Installation
 
 This project uses Go 1.21.
-Development and deployment are intended for a Linux environment,
-but all Go code is written in a platform independent fashion.
+Development and deployment are intended for Linux environments,
```

Comment:
This is not true anymore.
Development and deployment just rely on a POSIX system.